packages/waku/src/lib/builder/output-partykit.ts

```diff
@@ -0,0 +1,27 @@
+import path from 'node:path';
+import { existsSync, writeFileSync } from 'node:fs';
+
+import type { ResolvedConfig } from '../config.js';
+
+// XXX this can be very limited. FIXME if anyone has better knowledge.
```

Comment:
I think you would have better knowledge on partykit.

Reply:
a starter kit that already has this config would also work

Reply:
I mean, we might not need this comment in this file, which I added in output-cloudflare.ts.